app/views/paye/paye_total_income_tax.scala.html

```diff
@@ -0,0 +1,203 @@
+          <td id="higher-rate-income-tax-amount">@includes.currency_amount(viewModel.higherRateIncomeTaxAmount)</td>
+        </tr>
+          }
+
+        @if(!viewModel.additionalRateIncomeTax.isZero) {
+        <tr id="additional-rate-income-tax-row">
+          <td scope="row">@Html(Messages("ats.total_income_tax.table.additional",
+              """<span id="additional-rate-income-tax-before">"""+includes.currency_amount(viewModel.additionalRateIncomeTax)+"""</span>""",
+              """<span id="additional-rate-income-tax-rate">"""+viewModel.additionalRateIncomeTaxRate+"""</span>"""))</td>
+          <td id="additional-rate-income-tax-amount">@includes.currency_amount(viewModel.additionalRateIncomeTaxAmount)</td>
+        </tr>
+          }
+        }
+
+      </tbody>
+    </table>
+    }
+
+    @total_income_tax_includes.scottish_table(viewModel.scottishTax, viewModel.scottishRates)
+
+    @total_income_tax_includes.savings_table(viewModel.savingsTax, viewModel.savingsRates)
+
+    @if(viewModel.showDividendsTable) {
+    <table id="dividends">
+      <thead>
+        <tr id="dividends-section-row">
+          <th scope="col">@Html(Messages("ats.total_income_tax.dividends"))</th>
+          <th scope="col">@Html(Messages("generic.amount_pounds"))</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        @if(!viewModel.ordinaryRate.isZero) {
+        <tr id="ordinary-rate-row">
+          <td scope="row">@Html(Messages("ats.total_income_tax.table.basic",
+              """<span id="ordinary-rate-before">"""+includes.currency_amount(viewModel.ordinaryRate)+"""</span>""",
+              """<span id="ordinary-rate-rate">"""+viewModel.ordinaryRateTaxRate+"""</span>"""))</td>
+          <td id="ordinary-rate-amount">@includes.currency_amount(viewModel.ordinaryRateAmount)</td>
+        </tr>
+        }
+
+        @if(!viewModel.upperRate.isZero) {
+        <tr id="upper-rate-row">
+          <td scope="row">@Html(Messages("ats.total_income_tax.table.higher",
+              """<span id="upper-rate-before">"""+includes.currency_amount(viewModel.upperRate)+"""</span>""",
+              """<span id="upper-rate-rate">"""+viewModel.upperRateRate+"""</span>"""))</td>
+          <td id="upper-rate-amount">@includes.currency_amount(viewModel.upperRateAmount)</td>
+        </tr>
+        }
+
+        @if(!viewModel.additionalRate.isZero) {
+        <tr id="additional-rate-row">
+          <td scope="row">@Html(Messages("ats.total_income_tax.table.additional",
+              """<span id="additional-rate-before">"""+includes.currency_amount(viewModel.additionalRate)+"""</span>""",
+              """<span id="additional-rate-rate">"""+viewModel.additionalRateRate+"""</span>"""))</td>
+          <td id="additional-rate-amount">@includes.currency_amount(viewModel.additionalRateAmount)</td>
+        </tr>
+        }
+      </tbody>
+    </table>
+    }
+
+    @if(viewModel.incomeTaxStatus == "0002" && viewModel.restOfUkTotal.nonZero) {
```

Comment:
Is this the right check? Is it saying is a scottish tax payer and has money in the uk? it's a confusing statement